src/callbacks_stage/subcell_bounds_check.jl

```diff
@@ -0,0 +1,124 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+"""
+    BoundsCheckCallback(; output_directory="out", save_errors=false, interval=1)
+
+Bounds checking routine for [`SubcellLimiterIDP`](@ref). Applied as a stage callback for SSPRK
+methods. If `save_errors` is `true`, the resulting deviations are saved in
+`output_directory/deviations.txt` for every `interval` time steps.
```

Comment:
To me, this description does not tell me the purpose of this callback. That is, what is done with the information about the bounds check, or how is it used to control the scheme elsewhere?

As far as I can tell, it only saves the bounds checks in a variable, but that variable is never used anywhere, or is it?

Reply:
Yes, it is just a feature to give more information. The maximum error is always printed in the output at the end of the simulation. Then, it's up to the user to use this information.
And, as mentioned before, to better understand the appearance of errors, you can also export the errors from every time step if `save_errors=true`.

But I agree with you that the explanation is not very helpful if you are not very familiar with the idea of bounds for subcell limiting. I will revise the docstring.